src/createRollupPrepareTransactionRequest.ts

```diff
@@ -19,15 +26,22 @@ function createRollupEncodeFunctionData(args: CreateRollupFunctionInputs) {
   });
 }
 
+export type CreateRollupPrepareTransactionRequestParams = Prettify<
+  WithRollupCreatorAddressOverride<{
+    params: CreateRollupParams;
+    account: Address;
+    publicClient: PublicClient;
+    gasOverrides?: TransactionRequestGasOverrides;
+  }>
+>;
+
 export async function createRollupPrepareTransactionRequest({
   params,
   account,
   publicClient,
-}: {
-  params: CreateRollupParams;
-  account: Address;
-  publicClient: PublicClient;
-}) {
+  gasOverrides,
```

Comment:
Added `gasOverrides` for 2 reasons: 1) to make consistent with create token bridge and 2) to be able to set the gas limit to a constant in the unit test and skip gas estimation